packages/application-generic/src/tracing/index.ts

```diff
@@ -0,0 +1,35 @@
+import { OpenTelemetryModule } from 'nestjs-otel';
+export * from './tracing';
+
+export {
+  TraceService,
+  MetricService,
+  OtelInstanceCounter,
+  OtelUpDownCounter,
+  OtelHistogram,
+  OtelObservableGauge,
+  OtelObservableCounter,
+  OtelObservableUpDownCounter,
+  OtelCounter,
+  Span,
+} from 'nestjs-otel';
```

Comment:
I do like to have a single point of export for all the functionalities needed so there the swapping of the third party is quick. Might suggest to have a generic wrapper name that surrounds the functionality such as:
```
setCounter,
setHistogram,
setGauge,
```